cmd/apiserver/app/server/options.go

```diff
@@ -41,6 +41,7 @@ type ServiceCatalogServerOptions struct {
 	EtcdOptions *EtcdOptions
 	// TPROptions are options for serving with TPR as the backing store
 	TPROptions *TPROptions
+	StopCh     <-chan struct{}
```

Comment:
putting it inside `NewCommandServer` sounds good. we can always return the stop channel from that func later